httpobs/scanner/grader/grade.py

```diff
@@ -22,7 +22,28 @@
     0: 'F'
 }
 
+# See https://wiki.mozilla.org/Security/Standard_Levels for a definition of the risk levels
+# We cannot make an accurate decision on HIGH and MAXIMUM risk likelihood indicators with the current checks,
+# thus the likelihood indicator is currently at best (or worse) MEDIUM
+LIKELIHOOD_INDICATOR_CHART = {
+    'A+': 'LOW',
+    'A': 'LOW',
+    'A-': 'LOW',
+    'B+': 'MEDIUM',
+    'B': 'MEDIUM',
+    'B-': 'MEDIUM',
+    'C+': 'MEDIUM',
+    'C': 'MEDIUM',
+    'C-': 'MEDIUM',
+    'D+': 'MEDIUM',
+    'D': 'MEDIUM',
+    'D-': 'MEDIUM',
+    'F': 'MEDIUM'
+}
+
+
 GRADES = set(GRADE_CHART.values())
+LIKELIHOOD_INDICATOR = set(item for sublist in list(LIKELIHOOD_INDICATOR_CHART.values()) for item in sublist)
```

Comment:
You should already have the grade from get_grade_for_score; can you not just do a simple lookup based on that?